app/services/ccms/manual_review_determiner.rb

```diff
@@ -26,11 +26,17 @@ def initialize(legal_aid_application)
     end
 
     def call
-      return true if Setting.manually_review_all_cases? && non_passported?
+      needs_review?
+    end
+
+    private
 
-      return true if capital_contribution_required? && has_restrictions?
+    def needs_review?
+      manually_review_all_non_passported? || capital_contribution_required?
+    end
 
-      false
+    def manually_review_all_non_passported?
+      Setting.manually_review_all_cases? && non_passported?
     end
```

Comment:
What happens if there is no capital contribution required, but there are restrictions?  it should return true.